monkey/monkey_island/cc/services/reporting/report.py

```diff
@@ -155,9 +189,7 @@ def get_scanners_of_machine(cls, machine: Machine) -> List[Machine]:
 
     @classmethod
     def process_exploit_event(
-        cls,
-        exploitation_event: ExploitationEvent,
-        password_restored: Dict[ipaddress.IPv4Address, bool],
+        cls, exploitation_event: ExploitationEvent, password_restored: Dict[IPv4Address, bool]
```

Comment:
Password_restored is not Dict, it's defaultdict

Reply:
I think we should leave it as it is, since the method doesn't require or expect that `password_restored` be a `defaultdict`